test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -51,6 +51,11 @@ namespace test
 
 BOOST_FIXTURE_TEST_SUITE(SolidityEndToEndTest, SolidityExecutionFramework)
 
+int roundTo32 ( int _num )
+{
+	return ceil(_num / 32.0) * 32;
```

Comment:
Could just use `(num + 31) / 32`.

Also could make it `constexpr`.

Reply:
and it should probably go into devcore, turned into a template and named something like `smallestMultipleOf32`.

Reply:
And another two things learned